src/main/java/net/dv8tion/jda/api/JDA.java

```diff
@@ -1747,6 +1748,62 @@ default List<RichCustomEmoji> getEmojisByName(@Nonnull String name, boolean igno
         return getEmojiCache().getElementsByName(name, ignoreCase);
     }
 
+    /**
+     * Creates a new {@link ApplicationEmoji} for this bot.
+     *
+     * <p>Note that the bot is limited to 2000 Application Emojis (normal and animated).
+     *
+     * @param  name
+     *         The name for the new emoji (2-30 characters)
+     * @param  icon
+     *         The {@link Icon} for the new emoji
+     */
+    @Nonnull
+    @CheckReturnValue
+    RestAction<ApplicationEmoji> createApplicationEmoji(@Nonnull String name, @Nonnull Icon icon);
+
+    /**
+     * Retrieves a list of Application Emojis together with their respective creators.
+     *
+     * @return {@link RestAction RestAction} - Type: List of {@link ApplicationEmoji}
+     */
+    @Nonnull
+    @CheckReturnValue
+    RestAction<List<ApplicationEmoji>> retrieveApplicationEmojis();
+
+    /**
+     * Retrieves an application emoji together with its respective creator.
+     *
+     * @param  emojiId
+     *         The emoji id
+     *
+     * @throws IllegalArgumentException
+     *         If the provided id is not a valid snowflake
+     *
+     * @return {@link RestAction RestAction} - Type: {@link ApplicationEmoji}
+     */
+    @Nonnull
+    @CheckReturnValue
+    default RestAction<ApplicationEmoji> retrieveApplicationEmojiById(long emojiId)
+    {
+        return retrieveApplicationEmojiById(String.valueOf(emojiId));
+    }
+
+    /**
+     * Retrieves an application emoji together with its respective creator.
```

Comment:
Is creator the same as owner? We should use consistent terminology

Reply:
An application emoji is owned by the application, but the creator is the user in the dev-portal that created it.
Or in the case that the application itself creates it, then the application is the creator iirc

Reply:
I actually dont know what I should write instead.